projects/front_power_distribution/inc/front_power_distribution_gpio.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// Receive GPIO events and set the corresponding GPIO pin states.
+// Requires GPIO to be initialized.
+
+#include "event_queue.h"
+#include "gpio.h"
+
+void front_power_distribution_gpio_init(void);
+
+// Call in main event-processing loop.
```

Comment:
this comment isn't necessary... it's implied that `*process_event` functions are called in `main`. All of our codebase shows that.